src/SFA.DAS.FAT.Web.UnitTests/Controllers/CourseProvidersControllerTests/WhenGettingCourseProviders.cs

```diff
@@ -287,6 +294,58 @@ public async Task Then_Distance_Defaults_To_Ten_Miles(
         }
     }
 
+    [Test, MoqAutoData]
```

Comment:
Couple of cases needs coverage:
- A test for when location existed and order by is persisted to user's choice.
- A test for when location is removed and order by changes to default

Reply:
@DevenKShah I've added coverage for these, have a look